app/services/platform/encrypted_user_id_and_token.rb

```diff
@@ -7,9 +7,5 @@ def encrypted_user_id_and_token
         key: service_secret
       ).encrypt("#{user_id}#{session[:user_token]}")
```

Comment:
```suggestion
      ).encrypt("#{session[:user_id]}#{session[:user_token]}")
```

is the user_id method being moved somewhere else?

Reply:
oh sorry never mind, i just saw the alias